### ISSUE TITLE
Removed empty lines in multiline string literals

### ISSUE THREAD

**Issue by marciniwanicki** [nicklockwood/SwiftFormat#450]
It's more a question than a bug. I used SwiftFormat with default settings in one of my projects which is producing a lot of markdown content.

In unit tests I usually use multiline string literals (`"""`). In some cases SwiftFromat broke my tests by removing expected empty lines from the strings. 

Example:

```swift
let content = """


       # Header 1
       
       
       # Header2


       """
```

was converted to:

```swift
let content = """

       # Header 1
       
       # Header2

       """
```

I think it would be great if SwiftFromat can avoid code breaking changes, in this situation we could:

1. Leave new lines without any changes
2. Replace multiple new lines with a single new line and explicit `\n`

Curious what other thing about this case.

Happy to help with the PR if you think it deserves a patch.

**Comment by nicklockwood** [comment]
This is definitely a bug. I'll get it fixed asap.

**Comment by nicklockwood** [comment]
@marciniwanicki fixed in 0.40.10.
